vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,91 @@
+# Reconcile Mouse APIs in React Native - Desktop
+
+Windows and macOS support mouse devices today, but their React Native counterparts are not fully flushed out yet to unblock customers attempting to create React Native apps on those devices.
+
+This document will attempt to capture what's needed within React Native for Windows and macOS to enable customer scenarios. This includes:
+
+- Aligning with the [react-native-web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0) mouse behavior/implementation [issue TBD]()
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
+## `hovered`
```

Comment:
Is this prop only for `Pressable` or should this be introduced in `View` or others as well for completeness? What does web do? Perhaps @ necolas can help you figure this out.